packages/vscode-extension/src/webview/components/DeviceSettingsDropdown.tsx

```diff
@@ -117,7 +119,57 @@ function DeviceSettingsDropdown({ children, disabled }: DeviceSettingsDropdownPr
               </Slider.Root>
               <span className="device-settings-large-text-indicator" />
             </div>
+            <div className="device-settings-margin" />
+            {projectState.selectedDevice?.platform === DevicePlatform.IOS && (
+              <DropdownMenu.Sub>
+                <DropdownMenu.SubTrigger className="dropdown-menu-item">
+                  <span className="codicon codicon-layout" />
+                  Biometrics
+                  <span className="codicon codicon-chevron-right right-slot" />
+                </DropdownMenu.SubTrigger>
 
+                <DropdownMenu.Portal>
+                  <DropdownMenu.SubContent className="dropdown-menu-content">
+                    <DropdownMenu.Item
+                      className="dropdown-menu-item"
+                      onSelect={() => {
+                        project.updateDeviceSettings({
+                          ...deviceSettings,
+                          biometricEnrollment: !deviceSettings.biometricEnrollment,
+                        });
+                      }}>
+                      <span className="codicon codicon-layout-sidebar-left" />
+                      Enrolment
```

Comment:
This doesn't mean much to me, I'd preferably see here "Registered", "Configured" or similar.

Reply:
enrolled is used by ios simulator if there is a word that is easiest to understand for people how are using this feature it is this one

Reply:
![Screenshot 2024-08-28 at 11 43 31](https://github.com/user-attachments/assets/89b74681-4f1b-46f0-8fb7-5cba0d3d44e5)